templates/front/my_checks_desktop.html

```diff
@@ -77,7 +77,7 @@
         {% if check.last_ping %}
             <span
                 data-toggle="tooltip"
-                title="{{ check.last_ping }} GMT">
+                title="{{ check.last_ping|date:'N j, Y, P e' }}">
```

Comment:
taken from https://docs.djangoproject.com/en/1.9/ref/settings/#std:setting-DATETIME_FORMAT
with `e` added at the end to make sure we are explicit about which timezone